chain/jsonrpc/src/lib.rs

```diff
@@ -958,6 +976,21 @@ impl JsonRpcHandler {
     }
 }
 
+#[cfg(feature = "ganache")]
+impl JsonRpcHandler {
+    async fn ganache_patch_state(&self, params: Option<Value>) -> Result<Value, RpcError> {
+        let records = parse_params::<Vec<StateRecord>>(params)?;
```

Comment:
When designing RPC, it makes sense to think about forward compatibility. "If we are to add new parameter here, can we avoid breaking existing clients?". If what gets send on the wire is an array, then the answer is "no". For this reason it makes sense to introduce something like

```rust
#[derive(Serialize)]
pub struct RpcGanachePatchState {
    pub records: Vec<StateRecord>
}
```

A side benefit is that the API signature is clearly visible in the type defitniton, we don't need to hunt for it in method's implementation

Reply:
done